docs/components/rule.md

```diff
@@ -84,20 +84,24 @@ indicate connection, incompatibility or misconfiguration problems.
 either gap in rule or potentially ignored alert. Alert heavily on this if this happens for longer than your alert thresholds.
 `strategy` label will tell you if failures comes from rules that tolerate [partial response](rule.md#partial-response) or not.
 
-* `prometheus_rule_group_last_duration_seconds < prometheus_rule_group_interval_seconds`  If the difference is large, it means 
-that rule evaluation took more time than the scheduled interval. It can indicate that your query backend (e.g Querier) takes too much time 
-to evaluate the query, i.e. that it is not fast enough to fill the rule. This might indicate other problems like slow StoreAPis or 
-too complex query expression in rule. 
+* `prometheus_rule_group_last_duration_seconds < prometheus_rule_group_interval_seconds`  If the difference is large, it means
+that rule evaluation took more time than the scheduled interval. It can indicate that your query backend (e.g Querier) takes too much time
+to evaluate the query, i.e. that it is not fast enough to fill the rule. This might indicate other problems like slow StoreAPis or
+too complex query expression in rule.
 
 * `thanos_rule_evaluation_with_warnings_total`. If you choose to use Rules and Alerts with [partial response strategy's](rule.md#partial-response)
 value as "warn", this metric will tell you how many evaluation ended up with some kind of warning. To see the actual warnings
 see WARN log level. This might suggest that those evaluations return partial response and might not be accurate.
 
+* `prometheus_rule_group_last_duration_seconds > prometheus_rule_group_interval_seconds`. With all the networking overhead of thanos - you might end up in a situation when actual group evaluation
```

Comment:
Would argue if that `with all the networking overhead` comment is useful. You can end up in this situation for Prometheus as well. Also you might have local setup, so there might be not much of networking overhead, so it really depends.
 
```suggestion
* `prometheus_rule_group_last_duration_seconds > prometheus_rule_group_interval_seconds`.You might end up in a situation when actual group evaluation
```